packages/file-storage/tsup.config.ts

```diff
@@ -0,0 +1,20 @@
+import { defineConfig } from 'tsup';
+
+export default defineConfig([
+  // Platform-neutral build
+  {
+    clean: true,
+    dts: true,
+    entry: ['src/file-storage.ts', 'src/memory.ts'],
+    format: ['cjs', 'esm'],
+    platform: 'neutral',
+  },
+  // Node specific build
```

Comment:
I consider this build platform-neutral as well. The `node:fs` imports should be supported on all JS runtimes.

Reply:
The bundler doesn't consider them neutral. This "platform" flag really just determines default externals. By having the two builds we know if `node:*` imports end up in code that should just be operating on built-in globals.